themes/qgrid/layout/partials/header.ejs

```diff
@@ -5,6 +5,8 @@
 <% } else { %>
 <button class="nav-trigger" id="nav-trigger"></button>
 <a href="<%- url_for('/') %>" class="logo"></a>
+<button class="search-trigger" id="search-trigger"></button>
+<input id="search" class="search" placeholder="Search"/>
```

Comment:
type="search"

Reply:
I don't like default search type styles

Reply:
I don't like defauld Chrome and Safari styles for this type of input